### PR TITLE
Convert all pages to Markdown

### PR DESCRIPTION
**Pull request by Abbe98**
This replaces all HTML (except comments) with Markdown.

Notes:

 - Includes minor cleanup too(whitespace etc).
 - Width of tables has changes as it's now dynamic instead of being specified inline, a minor change was needed to display images nicely.

Tested locally in the browser but not in mapper itself.

### CODE REVIEW

File: pages/android-pc.md [hunk 1]
```diff
@@ -2,95 +2,42 @@
...
-objects, so Mapper's map display does not become slower. This may be relevant on
-mobile devices because of little processing power or because of battery runtime.
-
-If you want to use this functionality, you should best load a separate,
-initially transparent image template in png format for it. Suited images can be
-downloaded here:
-
- - <a href="attachment/scribble_1024.png">size 1024x1024</a>
- - <a href="attachment/scribble_2048.png">size 2048x2048</a>
+The Android app cannot be used to create new maps because it would be cumbersome to setup the base maps there. Instead, maps should be prepared with templates and possibly georeferencing using the PC version of Mapper first and then transferred to the mobile device for surveying.
```
Comment (dg0yt) [inline pages/android-pc.md:5]: Do we really dislike the fixed line breaks that much?

File: pages/colors_symbols.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ edited: 26 November 2015
 
 Modifying map colors and symbols
 
-[Color Window](color_dock_widget.md){: .subpage} <br />
+[Color Window](color_dock_widget.md){: .subpage}
```
Comment (dg0yt) [inline pages/colors_symbols.md:8]: I'm sure the `<br />` is there intentionally, for the Markdown converter in doxygen which builds the manual.

File: pages/file_menu.md [hunk 3]
```diff
@@ -10,72 +10,14 @@ edited: 29 November 2015
...
+| |**Open recent**| |This shows a list of recently opened files, recognizing that a file to be opened will often have been previously opened in the recent past.|
+|![ ](../mapper-images/save.png)|**Save**|**Ctrl+S**|Saves the currently opened file. OCD files will be saved in version 8 format.|
+| |**Save as...**|**Ctrl+Shift+S**|Shows the file dialog for choosing a file name and file format to save the currently opened map.|
+| |**Import...**| |Permits the import of maps or data in other formats. Currently the following formats are supported: .omap, .xmap, .ocd (version 6 to 11), .gpx, .osm, .dxf.|
+| |**Export as...**| |Permits the export of the currently opened map to another format. Currently pdf and raster image export are supported.|
+|![ ](../mapper-images/print.png)|**Print**|**Ctrl+P**|Shows the print dialog to print the currently opened map.|
+| |**Settings**| |Opens the [settings dialog](settings.md). (On OS X, the settings are in the usual place instead.)|
+|![ ](../mapper-images/close.png)|**Close**|**Ctrl+W, Ctrl+F4**|Closes the current program window.|
+| |**Exit**|**Ctrl+Q**|Exits the program, which will close all open map windows.|
```
Comment (dg0yt) [inline pages/file_menu.md:23]: Fine for now. On the long run, I would prefer to get rid of the table, like we did for some other pages.

**Review by dg0yt — COMMENTED** [review]
This time, there are many changes even to Markdown pages, and at least some are controversial.

File: pages/android-pc.md (hunk 1 above)
Comment (Abbe98) [reply pages/android-pc.md:5]: Feels like something that should be left for peoples text editor settings, when built with Karmadown it just ends up being weird HTML. Note that spaces was missing next to each line break(although the browser renders them as such).

File: pages/colors_symbols.md (hunk 2 above)
Comment (Abbe98) [reply pages/colors_symbols.md:8]: Something that needs to be tested(therefor my final paragraph). 

Usually Markdown parsers writes a `<br>` tag when there is two white spaces followed by a actual line break. Not used very often/not known about so assumed they should be replaced.

File: pages/file_menu.md (hunk 3 above)
Comment (Abbe98) [reply pages/file_menu.md:23]: I see why, although the pages where they have been replaced is a pain too. This is something that has to be tested in Doxygen.

File: pages/colors_symbols.md (hunk 2 above)
Comment (Abbe98) [reply pages/colors_symbols.md:8]: As far as I can tell from the Doxygen docs the syntax should be supported.

**Pull request merged**